package.json

```diff
@@ -143,27 +144,31 @@
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.16",
     "@reduxjs/toolkit": "^1.7.1",
-    "bootstrap": "^4.5.3",
+    "@tanstack/react-table": "^8.20.5",
+    "@testing-library/dom": "^10.4.0",
```

Comment:
Where is it used? Even if it's used, I think it should be under `devDependencies`

Reply:
I concur. Yeah, only needed @testing-library/react (see our Icomment below about Jest) and @testing-library/dom when installing RTL.

Reply:
Agree - updated 0d9e6c39